views/index.pug

```diff
@@ -0,0 +1,65 @@
+html
+  include partials/head
+  body
+    div.content
+      h1.title Ballantine Adult Fantasy
+      img(src="/img/unicorncolophon.jpg", alt="Ballantine Adult Fantasy - Unicorn Colophon")
+      h2 Documentation for API
+      div.description
+        p Instructions as to how to make proper use of the BAF API.
+        p Quick reference outside of the technical details: 
+        p The series ran from 1969-1974 and was edited by Lin Carter. 
+          | There are 18 books "pre-series" that are spiritually if not 
+          | technically published under the Ballantine Adult Fantasy label, 
+          | but many of these were later reprinted and emblazoned with 
+          | the official unicorn colophon. In the series proper, there 
+          | were a total of 65 books published, 
+          | and two "post-series", for a grand total of 85 all told.
+        p Pre-series 1-18. Proper series 19-83. Post-series 84-85.
+      ul
+        li.item GET /books
+        ul
+          li Route: localhost:8080/books
+          li Description: Returns a list of all books in the series.
+          br
+        li.item GET /books/{title}
+        ul
+          li Route: localhost:8080/books/{title}
+          li Description: Returns a book matching the title.
+          br
+        li.item GET /books/{author}
+        ul
+          li Route: localhost:8080/books/{author}
+          li Description: Returns the matching author's books
+          br
+        li.item POST /books
+        ul
+          li Route: localhost:8080/books
+          li Headers:
+          ul
+            li Content-Type: application/json
+            li Authorization Bearer YOUR_API_KEY_HERE
```

Comment:
Will you be requiring an API Key? Just heads up, we won't be spending class time on these.